src/Network/API/Mandrill/Templates/Types.hs

```diff
@@ -0,0 +1,68 @@
+{-# LANGUAGE DerivingVia     #-}
+{-# LANGUAGE StrictData      #-}
+{-# LANGUAGE TemplateHaskell #-}
+module Network.API.Mandrill.Templates.Types
+  ( -- * Getting templates.
+    GetTemplatesRq(..)
+  , gtrq_key
+  , gtrq_label
+  , Label(..)
+  , unLabel
+  -- * Template data
+  , TemplatesResponse(..)
+  , tres_slug
+  , tres_name
+  , tres_labels
+  , Slug(..)
+  , unSlug
+  , Name(..)
+  , unName
+  ) where
+
+import           Data.Aeson
+import           Data.Aeson.TH
+import           Data.String                (IsString)
+import qualified Data.Text                  as T
+import           Lens.Micro.TH              (makeLenses)
+
+import           Network.API.Mandrill.Types
+
+-- | Request to get templates
+data GetTemplatesRq = GetTemplatesRq
+    { _gtrq_key   :: MandrillKey -- ^ API Key
+    -- ^ An optional Template label (for searching)
+    , _gtrq_label :: Maybe Label -- ^ An optional Template label (for searching)
```

Comment:
CamelCase?

Reply:
Its a fork, so I kept it in the same style as before. Maybe we should just do a full refactor someday.